lib/utils/compose_ts.ts

```diff
@@ -1312,7 +1311,7 @@ async function pushAndUpdateServiceImages(
 			serviceImage.status = 'failed';
 			throw error;
 		} finally {
-			await afterEach(serviceImage, props);
+			afterEach(serviceImage, props);
```

Comment:
If you check how this function is called, you will realize the await here is necessary, so let's add the await back
```suggestion
			await afterEach(serviceImage, props);
```
and change the function args to:
```ts
	afterEach: (
		serviceImage: import('@balena/compose/dist/release/models').ImageModel,
		props: object,
	) => Promise<void>,
```